src/getTime.js

```diff
@@ -18,7 +18,10 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const matches = /\b[0]\d:[0-5]\d\b|[2][0-3]:[0-5]\d\b/;
+  const time = str.match(matches);
```

Comment:
matches это совпадения с английского, это результат вызова функции `match`, внимательнее
```suggestion
  const matches = str.match(matches);
```